Keas.Mvc/Controllers/KeysController.cs

```diff
@@ -62,6 +68,7 @@ public string GetTeam()
             {
                 _context.Keys.Add(key);
                 await _context.SaveChangesAsync();
+                await _eventService.TrackCreateKey(key, _securityService.GetUser().Result);
```

Comment:
@jcubbage here you should use await as well. never .Result, it's unreliable in an MVC environment.

`await _eventService.TrackCreateKey(key, await _securityService.GetUser());`